### PR TITLE
Update ngrok zap

### PR DESCRIPTION
**Pull request by adidalal**
Adds zap config for ngrok

Progress on #8202

### CODE REVIEW

File: Casks/ngrok.rb [hunk 1]
```diff
@@ -9,6 +9,6 @@
 
   binary 'ngrok'
 
-  # todo verify that this does not contain user-created content
-  # zap :delete => '~/.ngrok'
+  zap :delete => '~/.ngrok2/ngrok.yml',
```
Comment (vitorgalvao) [inline Casks/ngrok.rb]: Why not leave it as in the comment?

Comment (adityadalal924) [inline Casks/ngrok.rb]: ngrok was updated to version 2, and with that, the config structure changed

"ngrok's configuration file format has been redesigned to be simpler and expose more options. The configuration file for 2.0 is a simple YAML file read by default from $HOME/.ngrok2/ngrok.yml"

See: https://ngrok.com/whatsnew

Comment (vitorgalvao) [inline Casks/ngrok.rb]: To clarify, what I meant is, why not leave simply as `zap :delete => '~/.ngrok2`? Why remove the file and then the directory *only if it’s empty*?

Comment (adityadalal924) [inline Casks/ngrok.rb]: The user can have multiple configurations stored there, but needs to manually create any besides `ngrok.yml`, so I thought that would fall under "user-generated content"

Comment (vitorgalvao) [inline Casks/ngrok.rb]: Ah, no, we discussed that and came to the conclusion `zap` should remove everything.

Comment (adityadalal924) [inline Casks/ngrok.rb]: Okay, then just the delete should be fine to merge in

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 6295e11904aa5938642b3269a2cca45467558545. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**